plugins/SkyCultureMaker/src/ScmConstellation.cpp

```diff
@@ -59,30 +59,30 @@ void scm::ScmConstellation::setConstellation(const std::vector<CoordinateLine> &
 	updateTextPosition();
 }
 
-void scm::ScmConstellation::drawConstellation(StelCore *core, const Vec3f &color) const
+void scm::ScmConstellation::drawConstellation(StelCore *core, const Vec3f &lineColor, const Vec3f &nameColor) const
 {
 	StelPainter painter(core->getProjection(drawFrame));
 	painter.setBlending(true);
 	painter.setLineSmooth(true);
-	painter.setFont(constellationLabelFont);
+	painter.setFont(constellationNameFont);
 
 	bool alpha = 1.0f;
-	painter.setColor(color, alpha);
+	painter.setColor(lineColor, alpha);
 
 	for (CoordinateLine p : constellationCoordinates)
 	{
 		painter.drawGreatCircleArc(p.start, p.end);
 	}
 
-	drawNames(core, painter);
```

Comment:
The main change in this PR is this line: in some previous PR, the "nameColor" parameter was removed.